src/styles/MyStyles.js

```diff
@@ -12,6 +12,35 @@ const useMyStyles = makeStyles((theme) => ({
     minHeight: "40%",
     textTransform: "uppercase",
   },
+  grid__Page: {
+    width: "80%",
+  },
+  box__Page: {
+    backgroundColor: "#D8DDE1",
+    padding: "0 1rem 1rem 1rem",
+    width: "90%",
+    border: "1px solid black",
+    textAlign: "center",
+  },
+  card__TeamMember: {
+    marginBottom: ".1rem",
+  },
+  cardContent__TeamMember: {
+    display: "flex",
+    flexDirection: "row",
+    alignItems: "center",
+  },
+  cardBox__TeamMember: {
+    display: "flex",
+    flexDirection: "column",
+    flex: "0 0 30%",
+    paddingRight: "1em",
+    alignItems: "center"
+  },
+  teamMemberImage: {
+    width: "50%",
+    height: "50%"
```

Comment:
read about image scaling, would be better to scale it with for example max-width and auto height, but it's tricky :)

Reply:
Ok, I read about it and changed styling.